### ISSUE TITLE
Services/Switch for cam settings

### ISSUE THREAD

**Issue by SmartM-ui** [fuatakgun/eufy_security#63]
Hi,
would it be possible to implement services / switch to manage cam settings directly via Home Assistant?

It would be useful to be able to set the various options for motion detection, audio, etc., through Home Assistant.

Thanks

**Comment by fuatakgun** [comment]
do you any specific need or this is a general ask? there are many settings for cameras and I am not sure covering all here would be beneficial.

**Comment by SmartM-ui** [comment]
> do you any specific need or this is a general ask? there are many settings for cameras and I am not sure covering all here would be beneficial.

Hi,
it would be useful to have a control panel on Home Assistant without having to open the official app and, above all, to be able to take advantage of the automations.

If you think it appropriate to insert only a few, I try to write you the most useful for me.

Thanks again

**Comment by SmartM-ui** [comment]
> Give me some examples to initiate this, i will try to build it in a broader way.

It would be useful to have the switches for detecting motion and the type of objects to be detected, for audio detection and the switch for night vision

Thanks

**Comment by fuatakgun** [comment]
until we have a better solution, please take a look at here: https://github.com/fuatakgun/eufy_security/issues/71#issuecomment-934289771

**Comment by SmartM-ui** [comment]
> until we have a better solution, please take a look at here: [#71 (comment)](https://github.com/fuatakgun/eufy_security/issues/71#issuecomment-934289771)

I'll check right away, thanks!

**Comment by SmartM-ui** [comment]
Hi @fuatakgun ,
i tried to define eufy_security.send_message services and they seem to work fine!
Thanks for the examples.

I tried to create a switch for the led and it manages to manage it:
```
  - platform: template
    switches:
      eufy_led:
        turn_on:
          service: eufy_security.send_message
          data:
            message: >-
              {
                "messageId": "message_id_do_not_add_value_here", 
                "command": "device.set_status_led",
                "serialNumber": "XXX", 
                "value": true
              }
        turn_off:
          service: eufy_security.send_message
          data:
            message: >-
              {
                "messageId": "message_id_do_not_add_value_here", 
                "command": "device.set_status_led",
                "serialNumber": "XXX", 
                "value": false
              }
```
<img width="357" alt="Schermata 2021-10-05 alle 15 32 17" src="https://user-images.githubusercontent.com/63554131/136033067-78cf2945-fc8d-4238-94c5-875c3edaaa0b.png">


There is always the inconvenience that it cannot recover the real state of the Eufy app setting, but it is already excellent to have the possibility to interact with the cam from Home Assistant.

You could also think about creating the service (type eufy_security.enable or eufy_security.disable) so every 10 minutes it would be checked and reported the real status.

**Comment by fuatakgun** [comment]
Can i publish your sensors in my upcoming big change?

There are many architectural changes happened in my local setup, i will probably merge them soon. Please let me know if you specifically want to do the changes or i can include them directly?

**Comment by SmartM-ui** [comment]
Hi @fuatakgun ,
these are the binary_sensors that I put in my .py:

```
        ("motiontracking", "Motion Tracking", "motionTracking", "mdi:go-kart-track", None),
        ("notificationperson", "Notification Person", "notificationPerson", "mdi:bell-ring", None),
        ("notificationpet", "Notification Pet", "notificationPet", "mdi:bell-ring", None),
        ("notificationallothermotion", "Notification All Other Motion", "notificationAllOtherMotion", "mdi:bell-ring", None),
        ("notificationcrying", "Notification Crying", "notificationCrying", "mdi:bell-ring", None),
        ("notificationallsound", "Notification All Sound", "notificationAllSound", "mdi:bell-ring", None),
        ("audiorecording", "Audio Recording", "audioRecording", "mdi:record-rec", None),
        ("rtspstream", "RTSP Stream", "rtspStream", "mdi:cast-connected", None),
        ("speaker", "Speaker", "speaker", "mdi:bullhorn", None),
        ("microphone", "Microphone", "microphone", "mdi:microphone", None),
        ("autonightvision", "auto Night vision", "autoNightvision", "mdi:weather-night", None),
```

There is to check the speech of the status LED, which is important to me and which I was unable to import:
```
("statusled", "Status Led", "statusLed", "mdi:led-on", None),
```

Note that it displays and updates in real time, for example, in the binary_sensor.salotto_enabled only after the sensor has been enabled or disabled. 

Could you see if you can implement a service for selecting the sensors to be imported, such as the one present for the NUT, so that only the elements one needs are inserted? I attach screenshots:

<img width="321" alt="Schermata 2021-10-10 alle 17 25 25" src="https://user-images.githubusercontent.com/63554131/136702677-d5eb59d8-9104-4051-8566-395bffb218db.png">

Thanks

**Comment by SmartM-ui** [comment]
> I had added respective binary sensors and I liked the improvement, can you create a separate issue to track this request? Latest codebase has this sensors now.

Hi, 
what request do I need to open to continue tracking it?
The request for the statusLed or for the possibility to enable only the selected sensors?

**Comment by leonardpitzu** [comment]
any plan on adding the switches by default? it would be great!
use case: the interior cameras are turned off when i am at home though the IR LED (night mode) keeps toggling on/off depending on the light even tough it's useless as the camera is on and no streaming can take place.
